package.json

```diff
@@ -37,30 +37,25 @@
   "engines": {
     "node": ">=0.12.0"
   },
-  "peerDependencies": {
-    "pixi.js": "^5.0.3",
-    "react": "^16.6.3",
-    "react-bootstrap": "0.32.1",
-    "react-dom": "^16.6.3"
-  },
```

Comment:
I still don't understand why we're force bundling `react` and `react-dom` now. Maybe I am missing something but whenever you search for the purpose of `peerDependencies`, one finds `react` and `react-dom` [as](https://classic.yarnpkg.com/lang/en/docs/dependency-types/#toc-peerdependencies) [the](https://code-trotter.com/web/dependencies-vs-devdependencies-vs-peerdependencies/) [examples](https://stackoverflow.com/questions/30451556/what-is-the-correct-way-of-adding-a-dependency-to-react-in-your-package-json-for). If someone else wants to use HiGlass in their own React app they'll potentially end up with two copies of React in their final bundle. Also, as of npm v7 peer dependencies are installed automatically so there isn't even a convenience difference.

Reply:
Maybe it doesn't matter as the packages are defined as externals in webpack. I guess I would just find it cleaner if they were marked as `peerDependencies` as well but whatever.

Reply:
Alright. Added back.